bin/ci-import

```diff
@@ -2,6 +2,8 @@
 . bin/ci-process-bootstrap
 
 echo '{"write":"elife_search", "read":"elife_search"}' > index.json
+# FUTURE:
+# php bin/console keyvalue:store index-metadata '{"write":"elife_search", "read":"elife_search"}' --env="$environment"
```

Comment:
I spy a new library.

Reply:
Any project where it would be useful? I thought it would be a debug tool, mostly; targeting different backends than ElasticSearch?

Reply:
Journal's going to need a KV store soon (probably with a DB?). There are a couple of existing KV libraries, but they don't seem to be maintained.

Reply:
Ok, will extract after the search ordeal, when stable, with only the existing backend. Adding to ELPP-3180 checklist.